ibis/backends/clickhouse/__init__.py

```diff
@@ -548,22 +549,28 @@ def read_parquet(
         self,
         path: str | Path,
         table_name: str | None = None,
-        engine: str = "File(Parquet)",
+        engine: str = "File(Native)",
```

Comment:
This had to be changed to allow appending multiple files to the same table. `File(Parquet)` doesn't support this without a server side setting, which may not be configurable by the user.